ncolony/tests/test_process_events.py

```diff
@@ -128,6 +128,27 @@ def test_add_with_junk(self):
         self.assertEquals(env, {})
         self.assertEquals(self.logMessages, ['Added monitored process: hello'])
 
+    def test_add_with_inherited_env(self):
+        """Test a process addition with all the optional arguments"""
+        receiver = process_events.Receiver(self.monitor, dict(PATH='123'))
```

Comment:
Why not assign the environment dictionary to a local variable?  Then you wouldn't have to duplicate the dictionary in the assertion below.

Reply:
done (with a little flourish)